generative_models/VAE-GAN-tf/README.md

```diff
@@ -0,0 +1,38 @@
+# vae-gan-tf
+
+
+# Paper
```

Comment:
It should contain usage, help log, contributed by, and reference (change paper title to Reference). Also, see other readme for ref